### ISSUE TITLE
The strategy of transferring ImageNet-21k models to ImageNet-1K.

### ISSUE THREAD

**Issue by whai362** [Alibaba-MIIL/ImageNet21K#38]
Hi, I try to transfer the ImageNet-21k model (resnet50_miil_21k) to ImageNet-1K according to the details provided in your paper and https://github.com/Alibaba-MIIL/ImageNet21K/blob/main/Transfer_learning.md. But I only get 79.5% top-1 acc on ImageNet-1K, lower than the 82.0% in your readme. Could you give me some suggestions?

My training command is as follows:
python3 -m torch.distributed.launch --nproc_per_node=8 train.py \
data/ImageNet-1k/ \
-b=128 \
--img-size=224 \
--epochs=100 \
--color-jitter=0 \
--amp \
--sched='cosine' \
--model-ema --model-ema-decay=0.995 --reprob=0.5 --smoothing=0.1 \
--min-lr=1e-8 --warmup-epochs=3 --train-interpolation=bilinear --aa=v0 \
--pretrained \
--lr=2e-4 \
--model=resnet50 \
--opt=adam --weight-decay=1e-4

**Comment by mrT23** [comment]
i am not gonna provide code for every downstream dataset.

anyway, why do you think you are loading 'resnet50_miil_21k' pretrain ? it is not a part of timm. i think you are loading standard 1K ResNet50.
make sure you directly give it the proper pretrain file, and that the pretrain dimensions are ok

**Comment by whai362** [comment]
I am sure I loaded "resnet50_miil_21k" (https://miil-public-eu.oss-eu-central-1.aliyuncs.com/model-zoo/ImageNet_21K_P/models/resnet50_miil_21k.pth).
Except for the last FC used for image classification, all weights in "resnet50_miil_21k" have been loaded correctly.

The top-1 acc reported in the readme is 82.0%, but it seems no corresponding weights in this repo. 
![image](https://user-images.githubusercontent.com/8529570/126859848-f7027bc4-f254-4e5a-9e68-7db4d3c5cbe4.png)

I understand that it is unnecessary to provide code for every downstream dataset.
But ResNet50 on Imagenet-1K is a widely used setting in image classification tasks, so I hope you can provide more details about this.

**Comment by mrT23** [comment]
use SGD optimizer, not adam
try learning rates 2e-4 to 6e-4 ( I used 4e-4 with batch 256).
it should bring you at least to ~81.0%. KD gives another 1.0%

if not, triple-check for bugs in your code.

**Comment by whai362** [comment]
Many thanks for your suggestions!!

**Comment by mrT23** [comment]
@whai362  you are welcome to open a merge request (or just share a reproducible code) if SGD runs well for you
